packages/aws-cdk/test/api/test.stacks.ts

```diff
@@ -92,9 +92,12 @@ export = testCase({
         Type: 'AWS::CDK::Metadata',
         Properties: {
           Modules: `${require('../../package.json').name}=${require('../../package.json').version}`
-        }
+        },
+        Condition: 'CDKMetadataAvailable',
```

Comment:
Must only add this `Condition` under the same condition as emitting it (i.e., `region === UNKNOWN_REGION`), otherwise we will reference it when it hasn't been emitted.

Reply:
Which is why literally 3 lines below is a check that the condition does, indeed, exist.